pkg/notes/options/options.go

```diff
@@ -69,6 +65,18 @@ const (
 	GitHubToken                            = "GITHUB_TOKEN"
 )
 
+// Format is the format for the document.
+type FormatSpec string
```

Comment:
Please either remove this type (seems currently unused) or use it below in the `const` definitions.

Reply:
Done.